lib/perl/Genome/Model/Tools/Analysis/Coverage/MergeReadcounts.pm

```diff
@@ -6,23 +6,26 @@ use warnings;
 
 
 class Genome::Model::Tools::Analysis::Coverage::MergeReadcounts{
-    is => 'Command',
+    is => 'Command::V2',
     has => [
 	bam_files => {
 	    is => 'String',
 	    is_optional => 0,
+	    is_many => 1,
 	    doc => 'comma-separated list of bam files to grab readcounts from, Output columns will be appended in this order',
 	},
 	
 	variant_files => {
 	    is => 'String',
 	    is_optional => 0,
+	    is_many => 1,
 	    doc => 'coma-separated list of text files containing snvs in annotation format (1-based, first 5-cols = [chr, st, sp, ref, var]), all variant sites will be combined to a list',
```

Comment:
This isn't something you changed, but there is a typo here and in the doc for `variant_sources`--`comma` is short an `m` in both places.